src/pages/index.vue

```diff
@@ -57,7 +57,15 @@
             <PVColumn field="password" header="PASSWORD">
               <template #body="{ data }">
                 <div v-element-hover="() => (hasCopiedPassword = false)" class="flex flex-row items-center group/password">
-                  <span class="font-mono text-password-color text-[20px] mr-[8px]">{{ data.password }}</span>
+                  <span class="font-mono text-password-color text-[20px] mr-[8px]">
+                    <Highlighter
+                      v-if="findBy === 'PASSWORD'"
+                      highlight-class-name="font-bold underline"
+                      :text-to-highlight="data.password"
+                      :search-words="input.split(' ')"
+                    />
```

Comment:
Can you try making this into a reusable component?

Reply:
I was thinking about that, but what would it reduce here? It looks like it would reduce the repeat of the `highlight-class-name` but other than that it would seem like an extra wrapper?

Reply:
I'd imagine the component would have an API like this:

```vue
<TextHighlighter v-model="myField" :words="myArrOfWords" :enable="myComputedBoolean">
```

We abstract the library in the process